terraform/modules/cloud/aws/deploy/fleet/main.tf

```diff
@@ -125,7 +131,7 @@ resource "aws_launch_configuration" "spot-with-additional-storage" {
 }
 
 data "template_file" "spot_user_data" {
-  template = "${file("${path.module}/templates/spot_user_data.bash")}"
+  template = "${file("${path.module}/templates/${var.spot_user_data_file}")}"
```

Comment:
Why use two templates and two vars, when they point to the same file now?

Reply:
In case we need difference user data as before.